doc/ref_model/chapters/chapter08.rst

```diff
@@ -202,14 +202,31 @@ The API and Cloud Brokerage models are illustrated in the following diagrams:
+-  consumer of cloud services, the typical consideration is total cost of ownership as the consumption is to usually to support internal business operations: BSS/OSS systems;
 -  provider of cloud services, through operation of their own cloud or reselling of cloud services, in which case the typical consideration is margin (cost to offer services vs income received).
-
 These two stances will drive differing approaches to how a CSP would look to manage how it interacts within a Multi-Cloud environment.
 
+As a consumer of cloud to support internal Business operations and BSS/OSS the focus is on meeting the applications needs of organisation. Historically this came with need to operate support the applications infrastructure needs. The result was a split of CIO organisation into Delivery and Operations groups. At the same time that CIO application workloads have been moving to SaaS Cloud Providers, CTO Network Systems have been moving from running on dedicated infrastructure to being virtualised, example are IMS, 3GPP (4G & 5G) functions, IP Routers and Firewalls are being provided as VNFs and CNFs.  These network loads are now also being deployed on private CSP clouds as well as public clouds.
+
+As outlined in section "8.2.2 Telco Cloud", the result is that the CSP "network" is now an interconnected set of distributed NFVi's (Cloud Infrastructure) supported by different Cloud Providers, including the CSP, and, hence, the term "Hybrid Multi-Cloud", and the need for the CSP to be able to support and utilize this interconnected cloud is both inevitable and essential.
+
+So as a consumer and provider of Cloud Services, the CSP will continue to need to build and manage its own Cloud Infrastructure as well as provide: 
+
+- cloud orchestration solutions to orchestrate the utilisation of cloud services and capabilities from its own and other Cloud Providers;
+- network orchestration solutions to manage the interconnectivity across its own and other Cloud Provider networks. 
+
+The interactions for this are outlined in the "Multi-Cloud Interactions Model", however, to realise this, the CSP will need to adopt and sponsor a set of standards that are necessary to support these interactions. The identification of existing appliable standardas and gaps across the interations needs to be completed. As a first step in the following criterial for inclusion of a standard / technology is defined:
+
+-  Provide capabilities that are necessary to achieve hybrid multi-cloud vision and the multi-cloud interactions
+-  Are already mature Open Standards that have either been adopted or nurtured by recognised bodies with the telecommunications industry (e.g. ITU, ETSI, TMForum, GSMA, 3GPP, ISO and national Standards Organiations , (ANSI etc,) NIST)
+-  There are reference implementations or an active open source project/s or consortia providing implementations (CNCF. Open Infra)
```

Comment:
```suggestion
-  Specify reference implementations or an active open source project/s or consortia providing implementations (e.g., CNCF. Open Infra)
```

Reply:
@pgoyal01 I have updated wording to make it clearer and also update the list so it consistently has lower case starting word. Thank you for feedback.